CHANGELOG.md

```diff
@@ -27,12 +28,19 @@ on timeout, try increasing the value, for example, twice. Also note that the
 execution of commands with the `--await` flag and without an explicitly
 specified time period is now limited to 1 minute. This value can be changed with
 `--timeout` flag.
+
```

Comment:
yes, it does not render ok since i decided to add a list for that PR changes (i checked it one more time):
![image](https://github.com/nspcc-dev/neofs-node/assets/127229985/f20a8481-cb2e-4bb3-8630-3d8356da4d27)

Reply:
i want the `Updating` section to look like [this](https://github.com/notimetoname/neofs-node/blame/16ae494d5b07eccc78652b2c83e2a33f368f82cf/CHANGELOG.md#L138-L148) or like [this](https://github.com/nspcc-dev/neofs-node/blame/5ca26a877c1ad5c21b416a00d292a50bd40964c7/CHANGELOG.md#L68-L83)

Reply:
oh, i got you, fixed